File: haikan-parent/haikan-manager-web/src/main/webapp/js/service/securityService.js
app.service('securityService',function($http){

	//分页查询
	this.findPage=function(pageNum,pageSize){
		return $http.get('../security/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize);
	}
	
	//查询值班时间段列表
	this.selectTimeList=function(){
		return $http.get('../security/selectTimeList.do');
	}
	
	//增加报警对象信息
	this.add=function(entity){
		return $http.post('../security/add.do',entity);
	}
	
	//根据id查询报警对象信息
	this.findOne=function(id){
		return $http.get('../security/findOne.do?id='+id);
	}
	
	//修改报警对象信息
	this.update=function(entity){
		return $http.post('../security/update.do',entity);
	}
	
	//批量删除报警对象信息
	this.dele=function(ids){
		return $http.get('../security/delete.do?ids='+ids);
	}
	
});

File: haikan-parent/haikan-manager-interface/src/main/java/com/haikan/manager/service/SecurityService.java
package com.haikan.manager.service;

import java.util.List;
import java.util.Map;

import com.haikan.pojo.TbSecurity;

import entity.PageResult;

/**
 * 报警对象服务层接口
 * @author Administrator
 *
 */
public interface SecurityService {

	/**
	 * 分页查询
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 查询时间列表
	 * @return
	 */
	public List<Map> selectTimeList();
	
	/**
	 * 添加报警对象信息
	 * @param tbSecurity
	 */
	public void add(TbSecurity tbSecurity);
	
	/**
	 * 根据id查询报警对象信息
	 * @param id
	 * @return
	 */
	public TbSecurity findOne(Long id);
	
	/**
	 * 修改报警对象信息
	 * @param tbSecurity
	 */
	public void update(TbSecurity tbSecurity);
	
	/**
	 * 批量删除报警对象信息
	 * @param ids
	 */
	public void delete(Long[] ids);
}
